### PR TITLE
Adding new peering trobuleshooting page.

### PR DESCRIPTION
**Pull request by non-fungible-nelson**
## Pull request checklist

Use the following list to make sure your PR fits the Besu documentation quality standard.

### Before creating the pull request

Make sure that:

- [x] [All commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] You've read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Documentation).
- [ ] You've [previewed your changes locally](https://wiki.hyperledger.org/display/BESU/Preview+the+documentation).

## Describe the change

Adding peering FAQ

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] Github Actions workflow
- [ ] Build and QA tools configuration (for example, lint rules or Vale style)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] Read the Docs configuration
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [x] You've fixed any issues raised by the tests.
- [ ] You've [previewed your changes on Read the Docs](https://wiki.hyperledger.org/display/BESU/Preview+the+documentation)
  and added a [preview link](#preview).

### CODE REVIEW

**Comment by non-fungible-nelson** [comment]
@macfarla @pinges might be good for you to review and add any relevant points I missed or make corrections.

File: docs/public-networks/how-to/troubleshoot/troubleshoot-peering.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+**Q: What is the benefit of increasing the number of peers?**
+
+A: Increasing the number of max peers will not speed up Besu sync, as the bottleneck during sync is Disk IO and CPU.
+
+Bear in mind that Besu's peers are only used for initial sync and transaction gossip, neither of which affects attestation performance. It's the beacon node connectivity that controls how quickly your receive blocks and how attestations are published. Increasing Besu's peer count won't help with attestations and will increase the load on your node which may make things worse.
+
+## Metrics
+
+Make sure you are capturing metrics to gain insights into peer values over time.
```
Comment (macfarla) [inline docs/public-networks/how-to/troubleshoot/troubleshoot-peering.md:52]:
```suggestion
Capturing metrics will help to gain insights into peering behavior over time.
```

File: docs/public-networks/how-to/troubleshoot/troubleshoot-peering.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+To enable
```
Comment (macfarla) [inline docs/public-networks/how-to/troubleshoot/troubleshoot-peering.md:54]: are these instructions already specified elsewhere in the docs? would it be better to link there

File: docs/public-networks/how-to/troubleshoot/troubleshoot-peering.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+---
+description: How to troubleshoot Peering
+tags:
+  - public networks
+---
+
+# Troubleshooting Peering
+
+Besu can have trouble peering for a number of reasons. It may have to do with your network router, machine environment, and node configuration. If you are having trouble, [configuring your ports](../connect/configure-ports.md) is a good place to start. Additionally, learning how to [manage peers](../connect/manage-peers.md) will help in maintaining healthy peer counts for your node. Below we have some frequently asked questions that may help you.
```
Comment (macfarla) [inline docs/public-networks/how-to/troubleshoot/troubleshoot-peering.md:9]:
```suggestion
A number of factors can affect the ability of your node to find and maintain peers. Your network router, machine environment, and node configuration are all important. If you are having peering issues, [configuring your ports](../connect/configure-ports.md) is a good place to start. Additionally, learning how to [manage peers](../connect/manage-peers.md) will help in maintaining healthy peer counts for your node. Below we have some frequently asked questions that may help you.
```

File: docs/public-networks/how-to/troubleshoot/troubleshoot-peering.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+# Peering FAQ
+
+**Q: Why can’t I find enough peers to sync?**
+
+A: Multiple possible reasons:
```
Comment (macfarla) [inline docs/public-networks/how-to/troubleshoot/troubleshoot-peering.md:15]:
```suggestion
A: 
```

File: docs/public-networks/how-to/troubleshoot/troubleshoot-peering.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+- Poor hardware spec: not enough CPU, disk IOPS or bandwidth to handle all the peers
+- Your ports are not open in your firewall and/or router
+- Besu sending large numbers of DNS requests may cause issues, see [issue #4375.](https://github.com/hyperledger/besu/issues/4375)
+- It is normal for peers to connect and disconnect, especially soon after you start your node.
+- Try setting [`p2p-host`](../../reference/cli/options.md#p2p-host) to your external IP address to allow inbound connections
+- In proof of stake, we can set [`fast-sync-min-peers`](../../reference/cli/options.md#fast-sync-min-peers) to less than 5, for example 3.
```
Comment (macfarla) [inline docs/public-networks/how-to/troubleshoot/troubleshoot-peering.md:22]: I think we changed the default to 3 or 1 so this one may be no longer relevant

File: docs/public-networks/how-to/troubleshoot/troubleshoot-peering.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+- `-Xp2p-peer-lower-bound` can be set to a minimum number of peers
+- Deleting the node key (which is autogenerated in your data directory) may help with finding new peers. Two potential reasons for this:
+    1. Your node (identified by the address associated with this key) has been put onto other peers' bad peer lists for some reason.
+    2. Peer discovery is actually influenced by the value of the node key, it has to do with the node 'distance' in the [discovery algorithm.](https://github.com/ethereum/devp2p/wiki/Discovery-Overview#kademlia)
+
+The Prysm team has good documentation on [EL + CL peering](https://docs.prylabs.network/docs/prysm-usage/p2p-host-ip) that may help.
+
+**Q: How do I open/forward my ports?**
+
+A: If you’re behind NAT, you probably need to port forward in your router. You may also need to configure your firewall. Forward and open `30303` (if using default p2p port) for both UDP and TCP. If your router supports UPNP, you can setting the [`--nat-method`](../../reference/cli/options.md#nat-method) to `UPNPP2PONLY`. You can learn more about [NAT methods here.](../connect/specify-nat.md)
```
Comment (macfarla) [inline docs/public-networks/how-to/troubleshoot/troubleshoot-peering.md:34]:
```suggestion
A: If you’re behind NAT, you probably need to set up port forwarding in your router. You may also need to configure your firewall. Forward and open `30303` (if using default p2p port) for both UDP and TCP. If your router supports UPNP, you can setting the [`--nat-method`](../../reference/cli/options.md#nat-method) to `UPNPP2PONLY`. You can learn more about [NAT methods here.](../connect/specify-nat.md)
```

File: docs/public-networks/how-to/troubleshoot/troubleshoot-peering.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+**Q: What is the ideal number of peers for Besu?**
+
+A: Default max is 25. Increasing the number of peers increases the bandwidth, CPU and disk access Besu uses to respond to peers. A poor hardware spec may be the reason for low peer numbers. You will experience diminishing returns with a larger number of peers (> 100).
+
+**Q: What is the benefit of increasing the number of peers?**
+
+A: Increasing the number of max peers will not speed up Besu sync, as the bottleneck during sync is Disk IO and CPU.
+
+Bear in mind that Besu's peers are only used for initial sync and transaction gossip, neither of which affects attestation performance. It's the beacon node connectivity that controls how quickly your receive blocks and how attestations are published. Increasing Besu's peer count won't help with attestations and will increase the load on your node which may make things worse.
```
Comment (macfarla) [inline docs/public-networks/how-to/troubleshoot/troubleshoot-peering.md:48]:
```suggestion
Bear in mind that Besu's peers are only used for initial sync and transaction gossip, neither of which affects attestation performance. It's the beacon node connectivity that controls how quickly you receive blocks and how attestations are published. Increasing Besu's peer count won't help with attestations and will increase the load on your node which may make things worse.
```

**Review by alexandratran — APPROVED** [review]
Made some edits, LGTM.

**Pull request merged**